debian/control

```diff
@@ -9,6 +9,7 @@ Build-Depends: debhelper (>= 8),
                python3-all,
                python3-setuptools,
                python-nose, python3-nose, python-mock,
+               python-enum34 python3-enum34,
```

Comment:
added this for support of python 3.4 enums in earlier versions: https://pypi.python.org/pypi/enum34